### ISSUE TITLE
`defineVitestConfig` not respecting custom `configFile`

### ISSUE THREAD

**Issue by mattkatt** [danielroe/nuxt-vitest#118]
Im attempting to create a `nuxt.config` file for running tests (named `nuxt-test.config`), which contains additional configuration for running tests that I do not want in the build config. I am attempting to pass the name of the config file into `defineVitestConfig`, but when running vitest, it's still is using the default `nuxt.config` file, which is failing tests. If the appropriate configs are moved from the test config into the default then tests run fine.

Example error steps to duplicate:
1. Install `nuxt-vitest` into a fresh nuxt project
2. Create a new `nuxt-test.config` file containing the module import:
    ```
    export default defineNuxtConfig({
        modules: ['nuxt-vitest'],
    })
    ```
3. Create a `vitest.config` file using `defineVitestConfig`, passing in the following config:
    ```
    export default defineVitestConfig({
        configFile: 'nuxt-test.config',
    })
    ```
4. run vitest

This will cause vitest to error as it complains that the 'nuxt-vitest' module is not defined in 'nuxt.config'.

**Comment by danielroe** [comment]
Yes, the `configFile` option there is a _vite_ config file, rather than a nuxt one. Would you consider adding a local module which includes all your test config?

We could also support a custom nuxt config file but that's not that common in the rest of the Nuxt ecosystem.

**Comment by danielroe** [comment]
It's now possible to define nuxt configuration overrides in your vitest config via `test.environmentOptions.nuxt`. Does that solve this issue for you?
